ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh

```diff
@@ -85,7 +85,14 @@ pullSecret: >
 EOF
 fi
 
-yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "${SHARED_DIR}/install-config.yaml" - <<<"
+echo "Installing from initial release $RELEASE_IMAGE_LATEST"
+oc adm release extract -a "$pull_secret_path" "$RELEASE_IMAGE_LATEST" \
+  --command=openshift-install --to=/tmp
+
+version=$(/tmp/openshift-install version | grep 'openshift-install' | awk '{print $2}' | cut -d '.' -f 1,2)
+# Add vSphere credentials if the version is 4.15 or more
+if [[ $(echo -e "4.15\n$version" | sort -V | tail -n 1) == "$version" ]]; then
```

Comment:
Hi @mhanss, my 2 cents to help reduce the complexity of the condition:
~~~
version=$(/tmp/openshift-install version | grep 'openshift-install' | awk '{print $2}' | cut -d '.' -f 1,2 --output-delimiter='')
# Add vSphere credentials if the version is 4.15 or more
if [[ "${version}" -ge "415" ]]; then
~~~

Reply:
Changes have been implemented.